lib/vector-config/src/external/chrono.rs

```diff
@@ -18,3 +21,13 @@ where
         Ok(generate_string_schema())
     }
 }
+
+impl<TZ> ToValue for DateTime<TZ>
+where
+    Self: ToString,
+    TZ: TimeZone,
+{
+    fn to_value(&self) -> Value {
+        Value::String(self.to_string())
+    }
+}
```

Comment:
This is an example of a manual implementation we'd be able to get rid of with a blanket impl of `ToValue for T where T: Serialize`.